src/ExtendedView.js

```diff
@@ -1,28 +1,27 @@
 import './ExtendedView.css'
 
 function ExtendedView(props) {
-  const singleMovie = props.singleMovieData.find(movie => movie.id === parseInt(props.id))
-  console.log(singleMovie)
+
 
   return(
     <div className="extended-page">
```

Comment:
Three nested divs is cause for alarm. Are there any other elements you can reach for?